examples/experimental/freebsd.yaml

```diff
@@ -5,6 +5,12 @@ images:
   arch: "x86_64"
 - location: https://download.freebsd.org/ftp/snapshots/VM-IMAGES/14.0-CURRENT/aarch64/Latest/FreeBSD-14.0-CURRENT-arm64-aarch64.qcow2.xz
   arch: "aarch64"
+- location: https://download.freebsd.org/ftp/snapshots/VM-IMAGES/14.0-CURRENT/riscv64/Latest/FreeBSD-14.0-CURRENT-riscv-riscv64.qcow2.xz
+  arch: "riscv64"
+  kernel:
```

Comment:
this u-boot image was mostly a hack to get past the validate (for amd64/arm64), but it _is_ the real image:

https://wiki.freebsd.org/riscv/QEMU

We might want to import it to https://github.com/lima-vm/u-boot-qemu-mirror/ proper, so it can be used here.

Reply:
Why can't we use Debian's u-boot binary for FreeBSD?

Reply:
I think that should be fine as well, just haven't tested it.